### PR TITLE
Indent contents of docstrings when they're reindented

### PR DESCRIPTION
**Pull request by vladmos**

### CODE REVIEW

**Comment by laurentlb** [comment]
All tests have a `def` inside a `if`.
Can we have tests with just a `def` (which is the case we care about)?

**Comment by vladmos** [comment]
Added more tests.

File: build/rewrite.go [hunk 1]
```diff
@@ -888,3 +896,70 @@ func (args loadArgs) Less(i, j int) bool {
...
+					break
+				}
+			}
+		}
+		if i != len(lines)-1 && isAllSpaces(line) {
+			// Replace a line that contains only spaces with an empty line
+			// unless it's the last line that's responsible for the indentation of the closing """
+			line = ""
+		}
+		lines[i] = line
```
Comment (laurentlb) [inline build/rewrite.go]: As we're processing the string, we could remove trailing whitespace at the same time.

File: build/rewrite.go [hunk 2]
```diff
@@ -888,3 +896,70 @@ func (args loadArgs) Less(i, j int) bool {
...
 }
+
+// indentDocstrings modifies the value of docstrings if their indentation level has been changed
+func indentDocstrings(f *File, info *RewriteInfo) {
+	Walk(f, func(v Expr, stk []Expr) {
+		def, ok := v.(*DefStmt)
+		if !ok {
+			return
+		}
+		if len(def.Body) == 0 {
```
Comment (laurentlb) [inline build/rewrite.go]: Merge both `if`

Comment (vladmos) [reply build/rewrite.go]: Done.

File: build/rewrite.go (hunk 1 above)
Comment (vladmos) [reply build/rewrite.go]: Done. If I do that, should I also modify docstrings that are not being indented to remove trailing spaces from their lines? `"indentdocstrings"` will need to be renamed to `"formatdocstrings"` in that case.

Comment (vladmos) [reply build/rewrite.go]: Remove trailing space from all docstrings for consitency.

**Pull request merged**